netbox/dcim/api/views.py

```diff
@@ -327,6 +327,13 @@ class DeviceViewSet(CustomFieldModelViewSet):
     )
     filterset_class = filters.DeviceFilter
 
+    _method = Parameter(
```

Comment:
Let's avoid using a generic variable name on DeviceView for something that's specific to the NAPALM endpoint. Honestly, I'd be okay with just defining the Parameter list directly inside the `@swagger_auto_schema()` decorator.

Reply:
Moved parameter to decorator